Symfony/CS/Fixer/Contrib/PhpdocToCommentFixer.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Ceeram <ceeram@cakephp.org>
+ */
+class PhpdocToCommentFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens->findGivenKind(T_DOC_COMMENT) as $index => $token) {
+            $nextIndex = $tokens->getNextMeaningfulToken($index);
```

Comment:
There may be no tokens after T_DOC_COMMENT.

Instead of removing `if` statement I wasn't covered by tests here (https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/969#discussion_r22883661) please add a test for it (and recover that `if`)